src/DeviceProvisioning/ProvisioningWebApi/ProvisioningWebApi.ConsoleHost/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Practices.IoTJourney;
+using Microsoft.Practices.IoTJourney.DeviceProvisioningModels;
+using Microsoft.ServiceBus;
+using Microsoft.ServiceBus.Messaging;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using CommonConsoleHost = Microsoft.Practices.IoTJourney.Tests.Common.ConsoleHost;
+
+namespace ProvisioningWebApi.ConsoleHost
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            AsyncPump.Run(() => MainAsync(args));
```

Comment:
We've removed AsyncPump.Run from the other console apps. Please update this console app.